javaparser-symbol-solver-core/src/main/java/com/github/javaparser/symbolsolver/resolution/MethodResolutionLogic.java

```diff
@@ -637,6 +637,16 @@ private static boolean areOverride(MethodUsage winningCandidate, MethodUsage oth
         throw new UnsupportedOperationException(typeDeclaration.getClass().getCanonicalName());
     }
 
+    public static SymbolReference<ResolvedMethodDeclaration> solveMethodInFQN(String fqn, String name,
+                                                                             List<ResolvedType> argumentsTypes, boolean staticOnly, TypeSolver typeSolver) {
+        SymbolReference<ResolvedReferenceTypeDeclaration> typeRef = typeSolver.tryToSolveType(fqn);
+        if (typeRef.isSolved()) {
+            return solveMethodInType(typeRef.getCorrespondingDeclaration(), name, argumentsTypes,
+                    staticOnly, typeSolver);
+        }
```

Comment:
I would throw an exception if the type is not solved

Reply:
Why would you throw an exception and not an unsolved symbol reference?

Reply:
Well, because we have not the possibility to check it properly: we are not saying that the method could not be found, we are saying that we are unable to check it. Or worse, we could solve it in the wrong way. For example, if a given method can be solved in a superclass and in an interface but we cannot find the superclass, so we solve it in an interface that would lead to the wrong result.

Reply:
This method is not used anymore, should I still keep it around and add the exception? Or just delete it entirely?

Reply:
Let's delete it. Thank you for your patience and your work on this one!